### PR TITLE
"Submit your app" updates

### PR DESCRIPTION
**Pull request by aw-asana**
Updates to the app submission process.

### CODE REVIEW

**Comment by aw-asana** [comment]
Screen Shot 2022-07-26 at 4 26 23 PM
Screen Shot 2022-07-26 at 4 26 06 PM
Screen Shot 2022-07-26 at 4 25 51 PM

File: source/includes/markdown/_14-publish-your-app.html.md [hunk 1]
```diff
@@ -68,28 +68,49 @@ The explorer is not meant to be an exhaustive tool with every endpoint in the AP
...
 # Submit your App
 
-<span class="description">
-Have you built a web application that you want to share with the Asana community? Submit [this form](https://www.asana.com/apps/submit) to get your app listed in the [Asana apps directory](https://www.asana.com/apps).
-</span>
+If you've built an application that you wish to share with the Asana community, you can visit the
+[developer console](https://app.asana.com/0/my-apps) to begin the app submission process. 
+
+By submitting an app for review, you'll have the opportunity to distribute your app publicly by listing
+it in the [app directory](https://asana.com/apps) (as well as in the in-product [app gallery](/docs/installing-an-app)
```
Comment (jv-asana) [inline source/includes/markdown/_14-publish-your-app.html.md:75]: `/docs/installing-an-app` looks like a dead link I tried to go to `/docs/installing-an-app` but it didn't take me anywhere

File: source/includes/markdown/_14-publish-your-app.html.md [hunk 2]
```diff
@@ -68,28 +68,49 @@ The explorer is not meant to be an exhaustive tool with every endpoint in the AP
...
+for apps with [App Components](/docs/overview-of-app-components)).
+
+### Submit for review
 
-### Recommendations and resources for app developers:
+When managing your app in the [developer console](https://app.asana.com/0/my-apps), select **Submit for review**
+in the sidebar to access the app submission form:
 
-* We strongly recommend that you use [OAuth 2.0](/docs/oauth) for your app. Apps that don’t use Auth 2.0 will likely not be approved for listing in our directory.
+<img src="../images/submit-my-test-app.png" />
```
Comment (jv-asana) [inline source/includes/markdown/_14-publish-your-app.html.md:83]: Do we care about `alt` text for screen readers?

File: source/includes/markdown/_14-publish-your-app.html.md [hunk 3]
```diff
@@ -68,28 +68,49 @@ The explorer is not meant to be an exhaustive tool with every endpoint in the AP
...
-* Use end-to-end encryption. Use a trusted site to test your TLS security (such as: https://www.ssllabs.com/ssltest/).
+Along with building or providing the above features, you'll be able to submit additional app details through the submit form,
+such as setup instructions, a screencast video link, and other listing details.
 
-* Join the [Asana developer community](https://forum.asana.com/c/developersAPI). This is the best resource to get technical questions answered as well as get notified about new API features, deprecations, and other breaking changes. 
+### App review process
 
-* Once your app is listed, you can answer Asana user questions in the [integrations section of the Asana community](https://forum.asana.com/c/integrations).
+Once your app has been submitted through the [developer console](https://app.asana.com/0/my-apps), the review process will
+begin. At this point, the the app will no longer be open for further edits. Note that if your app uses
```
Comment (jv-asana) [inline source/includes/markdown/_14-publish-your-app.html.md:102]: There's 2 `the` together

File: source/includes/markdown/_14-publish-your-app.html.md [hunk 4]
```diff
@@ -68,28 +68,49 @@ The explorer is not meant to be an exhaustive tool with every endpoint in the AP
...
+[App Components](/docs/overview-of-app-components), your submission will go through a security review and QA process
+before it’s listed in the in-product [app gallery](/docs/installing-an-app).
```
Comment (jv-asana) [inline source/includes/markdown/_14-publish-your-app.html.md:104]: Same issue here `/docs/installing-an-app` doesn't take you anywhere

File: source/includes/markdown/_14-publish-your-app.html.md [hunk 5]
```diff
@@ -68,28 +68,49 @@ The explorer is not meant to be an exhaustive tool with every endpoint in the AP
...
-### Updating your existing app listing
-If you wish to update your existing listing, send an email to api-support at asana.com with the specific changes. 
+You can track the status of your app submission on the same **Submit for review** page. If your app submission passes requirements,
+you'll receive an email notification that your app is ready to publish. At this point, you can select the **Publish app** button in 
+the developer console to share your app:
 
-We expect existing apps to maintain the same level of user experience that they had when accepted to the apps directory.  
+<img src="../images/approved-app.png" />
+
+If your app submission doesn't meet requirements, you will receive communications regarding the next steps (i.e., any changes required)
```
Comment (jv-asana) [inline source/includes/markdown/_14-publish-your-app.html.md:112]: Do we want to be more specific about the communication method? Like via email?

File: source/includes/markdown/_14-publish-your-app.html.md (hunk 1 above)
Comment (aw-asana) [reply source/includes/markdown/_14-publish-your-app.html.md:75]: `/docs/installation-flow` is the correct one

File: source/includes/markdown/_14-publish-your-app.html.md (hunk 4 above)
Comment (aw-asana) [reply source/includes/markdown/_14-publish-your-app.html.md]: Done!

File: source/includes/markdown/_14-publish-your-app.html.md (hunk 5 above)
Comment (aw-asana) [reply source/includes/markdown/_14-publish-your-app.html.md:112]: Sounds good to me

**Review by jv-asana — APPROVED** [review]
LGTM

**Pull request merged**